internal/app/machined/pkg/system/services/kubelet.go

```diff
@@ -290,7 +290,6 @@ func (k *Kubelet) args(r runtime.Runtime) ([]string, error) {
 		"container-runtime":          "remote",
 		"container-runtime-endpoint": "unix://" + constants.CRIContainerdAddress,
 		"config":                     "/etc/kubernetes/kubelet.yaml",
-		"dynamic-config-dir":         "/etc/kubernetes/kubelet",
```

Comment:
Wasn't used anywhere AFAIU

Reply:
Not by us by default. This was something users did on their own. Still safe to remove.